packages/block-editor/src/hooks/layout.js

```diff
@@ -130,15 +130,26 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 			<InspectorControls>
 				<PanelBody title={ __( 'Layout' ) }>
 					{ showInheritToggle && (
-						<ToggleControl
-							label={ __( 'Inherit default layout' ) }
-							checked={ !! inherit }
-							onChange={ () =>
-								setAttributes( {
-									layout: { inherit: ! inherit },
-								} )
-							}
-						/>
+						<>
+							<ToggleControl
+								label={ __( 'Blocks use full width' ) }
+								checked={ ! inherit }
+								onChange={ () =>
+									setAttributes( {
+										layout: { inherit: ! inherit },
+									} )
+								}
+							/>
+							<p className="block-editor-hooks__layout-controls-helptext">
+								{ !! inherit
+									? __(
+											'Nested blocks use default content width with options for full and wide widths.'
```

Comment:
Good question! Block themes can switch off wide/full alignments, but only by removing the `layout` section in their theme.json settings altogether, which means they're not able to set content size/wide size either. I somehow don't see that being a popular choice, but if it is done, then the toggle doesn't show:

<img width="281" alt="Screen Shot 2022-06-27 at 11 13 26 am" src="https://user-images.githubusercontent.com/8096000/175842983-5a5721f6-4e61-4329-a720-17509f62c834.png">

Reply:
Perfect 👍

> I somehow don't see that being a popular choice

Me either, but good to know what happens just in case someone tries to get rid of it altogether!